webserver/views/api.py

```diff
@@ -1,13 +1,20 @@
 from __future__ import absolute_import
-from flask import Blueprint, request, jsonify
+from flask import Blueprint, request, Response
 from webserver.decorators import crossdomain, ip_filter
 from werkzeug.exceptions import BadRequest, NotFound
 import db.data
 import db.exceptions
-import json
+import ujson
 
 api_bp = Blueprint('api', __name__)
 
+def ujsonify(*args, **kwargs):
+    """An implementation of flask's jsonify which uses ujson
+    instead of json. Doesn't have as many bells and whistles
+    (no indent/separator support).
+    """
+    return Response((ujson.dumps(dict(*args, **kwargs)), '\n'),
+                        mimetype='application/json')
 
```

Comment:
Ideally yes, but I think it is too early when we have such a simple function as the only shared component. Once we get more, we should do this though.